index.html

```diff
@@ -100,43 +89,48 @@
 }
 
     </script>
+
+    <!-- Use Nimiq's brand font (regular and bold weights). -->
+    <link href="https://fonts.googleapis.com/css?family=Muli:400,700" rel="stylesheet">
 </head>
 <body>
-    <h1>Getting started with the Nimiq API</h1>
-    This html file demonstrates the most simple way to build web applications on top of the <a target="_blank" href="https://nimiq.com">Nimiq Blockchain</a>.
+    <h1>Getting started with the Nimiq Client API</h1>
+    This HTML file demonstrates the most simple way to build web applications on top of the <a target="_blank" href="https://nimiq.com">Nimiq Blockchain</a>.
     <h2>Most Simple Client</h2>
-    <p id="client-selector">Client type: <a href="#" client-type="nano">nano</a> <a href="#" client-type="light">light</a></p>
+    <p id="client-selector">
+        Client type:
+        <a href="#" client-type="pico">pico</a>
+        <a href="#" client-type="light">light</a>
+    </p>
     <div id="client-info" style="display:none">
         <p>Status: <span id="message">Loading Nimiq.</span></p>
         <p>Account address: <span id="address"><em>loading</em></span></p>
-        <p>Balance: <img src="nimiq_white.svg" class="nimiq_currency"> <span id="balance"><em>loading</em></span></p>
+        <p>Balance: &#x2B23; <span id="balance"><em>loading</em></span></p>
         <p>Current block height: <span id="height"><em>loading</em></span></p>
         <p>Current number of peers: <span id="peers"><em>loading</em></span></p>
     </div>
 
     <h2>Source Code</h2>
     <a target="_blank" href="https://github.com/nimiq-network/nimiq-demo/blob/master/index.html">View Source Code on Github.</a>
+
     <style>
         body {
-            background:teal;
-            color:white;
-            font-family: sans-serif;
-            padding: 8%;
+            background: #F4F4F4;
+            color: #1F2348; /* Nimiq Indigo */
+            font-family: 'Muli', sans-serif;
+            padding: 6% 8%;
```

Comment:
Add Nimiq Style?

Reply:
Not necessary for only this little bit of text. This is not supposed to be a Nimiq-ecosystem demo. Only a quick blockchain-in-the-browser demo.